### ISSUE TITLE
Phone validator accept short phone numbers

### ISSUE THREAD

**Issue by Djameleddine21** [JamalBelilet/regexed_validator#3]
issue in the phone number validator,
i think this is the right regular expression :   r"^(?:\+?)(?:[0-9] ?){6,14}[0-9]$"

**Comment by manikandan-selvanathan** [comment]
It accepts long phone numbers too.

**Comment by Djameleddine21** [comment]
Yeah of course because there are a long number in some countries, so for that i devided the number on 3 parts :
'+' : optional
numbers : of zone ( t think you need to limit this part) 
6 to 14 : you can change it, but when is search in found those borns of any number in the world

**Comment by manikandan-selvanathan** [comment]
Agree. I agree the regex expression is invalid I believe. it accepts less and long numbers too. It should be restricted to max 15 numbers.
